<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>939</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="login_Page">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>100</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">Qlabel{
font-size:75px;
}</string>
            </property>
            <property name="text">
             <string>Bank al LUMS</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame">
            <property name="minimumSize">
             <size>
              <width>170</width>
              <height>100</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>100</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::Shape::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
            <widget class="QLabel" name="username_Label">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>171</width>
               <height>31</height>
              </rect>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">Qlabel{
font-size:15px;
}</string>
             </property>
             <property name="text">
              <string>UserName:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
             </property>
            </widget>
            <widget class="QLineEdit" name="lineEdit">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>40</y>
               <width>171</width>
               <height>28</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit{
font-size:15;
padding:5px;
}</string>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_2">
            <property name="minimumSize">
             <size>
              <width>170</width>
              <height>100</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>100</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::Shape::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
            <widget class="QLineEdit" name="lineEdit_2">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>40</y>
               <width>171</width>
               <height>28</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit{
font-size:15;
padding:5px;
}</string>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::EchoMode::Password</enum>
             </property>
            </widget>
            <widget class="QLabel" name="password_Label">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>10</y>
               <width>171</width>
               <height>31</height>
              </rect>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">Qlabel{
font-size:15px;
}</string>
             </property>
             <property name="text">
              <string>Password:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="login_button">
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
font-size:15;
background-color: qradialgradient(spread:reflect, cx:0.5, cy:0.5, radius:1.026, fx:0.331, fy:0.245, stop:0 rgba(4, 7, 255, 255), stop:1 rgba(255, 255, 255, 255));
}
</string>
            </property>
            <property name="text">
             <string>Login</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="createAccount_Button">
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
font-size:15;
background-color: qradialgradient(spread:reflect, cx:0.5, cy:0.5, radius:1.026, fx:0.331, fy:0.245, stop:0 rgba(4, 7, 255, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
            </property>
            <property name="text">
             <string>Create New account</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="receipt_Page"/>
      <widget class="QWidget" name="page_2"/>
      <widget class="QWidget" name="accounts_Page">
       <widget class="QPushButton" name="default_acc_button">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>400</y>
          <width>171</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string>Default Account</string>
        </property>
       </widget>
       <widget class="QPushButton" name="current_acc_button">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>140</y>
          <width>171</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string>Current Account</string>
        </property>
       </widget>
       <widget class="QPushButton" name="savings_acc_button">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>270</y>
          <width>171</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string>Savings Account</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="dashboard_Page">
       <widget class="QPushButton" name="amount1000_Button">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>80</y>
          <width>141</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string>1000</string>
        </property>
       </widget>
       <widget class="QPushButton" name="amount5000_Button">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>210</y>
          <width>141</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string>5000</string>
        </property>
       </widget>
       <widget class="QPushButton" name="amount20000_Button">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>470</y>
          <width>141</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string>20,000</string>
        </property>
       </widget>
       <widget class="QPushButton" name="amount10000_Button">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>340</y>
          <width>141</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string>10000</string>
        </property>
       </widget>
       <widget class="QPushButton" name="deposit_Button">
        <property name="geometry">
         <rect>
          <x>740</x>
          <y>340</y>
          <width>141</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string>Deposit</string>
        </property>
       </widget>
       <widget class="QPushButton" name="checkBalance_Button">
        <property name="geometry">
         <rect>
          <x>740</x>
          <y>470</y>
          <width>141</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string>Check Balance</string>
        </property>
       </widget>
       <widget class="QPushButton" name="otherAmount_Button">
        <property name="geometry">
         <rect>
          <x>740</x>
          <y>210</y>
          <width>141</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string>Other Amount</string>
        </property>
       </widget>
       <widget class="QPushButton" name="amount25000_Button">
        <property name="geometry">
         <rect>
          <x>740</x>
          <y>80</y>
          <width>141</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string>25,000</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="createUser_Page">
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>289</x>
          <y>60</y>
          <width>221</width>
          <height>461</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_3">
           <property name="frameShape">
            <enum>QFrame::Shape::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Shadow::Raised</enum>
           </property>
           <widget class="QLineEdit" name="newUser_Line">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>70</y>
              <width>181</width>
              <height>31</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="newUser_Line_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>40</y>
              <width>181</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>Enter UserName</string>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_5">
           <property name="frameShape">
            <enum>QFrame::Shape::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Shadow::Raised</enum>
           </property>
           <widget class="QLineEdit" name="confirmPassword_Line">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>70</y>
              <width>181</width>
              <height>31</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="confirmPassword_Label">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>40</y>
              <width>181</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>Confirm Password</string>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_4">
           <property name="frameShape">
            <enum>QFrame::Shape::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Shadow::Raised</enum>
           </property>
           <widget class="QLineEdit" name="newPassword_Line">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>70</y>
              <width>181</width>
              <height>31</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="newPassword_Label">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>40</y>
              <width>181</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>Enter Password</string>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="submitApplication_Button">
           <property name="styleSheet">
            <string notr="true">background-color: qradialgradient(spread:reflect, cx:0.5, cy:0.5, radius:1.026, fx:0.331, fy:0.245, stop:0 rgba(4, 7, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
           </property>
           <property name="text">
            <string>Create account</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="numpad_Page">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdNumber"/>
        </item>
        <item>
         <layout class="QGridLayout" name="numpad_Grid">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <item row="1" column="0">
           <widget class="QPushButton" name="four_Button">
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>75</height>
             </size>
            </property>
            <property name="text">
             <string>4</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QPushButton" name="cancel_Button">
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>75</height>
             </size>
            </property>
            <property name="text">
             <string>Cancel</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QPushButton" name="depositEnter_Button">
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>75</height>
             </size>
            </property>
            <property name="text">
             <string>Enter</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="blank_Button_2">
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>75</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QPushButton" name="blank_Button">
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>75</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="six_Button">
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>75</height>
             </size>
            </property>
            <property name="text">
             <string>6</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="five_Button">
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>75</height>
             </size>
            </property>
            <property name="text">
             <string>5</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="clear_Button">
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>75</height>
             </size>
            </property>
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="two_Button">
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>75</height>
             </size>
            </property>
            <property name="text">
             <string>2</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="three_Button">
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>75</height>
             </size>
            </property>
            <property name="text">
             <string>3</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QPushButton" name="blank_Button_3">
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>75</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="one_Button">
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>75</height>
             </size>
            </property>
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="nine_Button">
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>75</height>
             </size>
            </property>
            <property name="text">
             <string>9</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="eight_Button">
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>75</height>
             </size>
            </property>
            <property name="text">
             <string>8</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="zero_Button">
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>75</height>
             </size>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="seven_Button">
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>75</height>
             </size>
            </property>
            <property name="text">
             <string>7</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
